src/components/editor/ArticleListController.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { browserHistory } from 'react-router';
+import FalcorController from "lib/falcor/FalcorController";
+import List from "./List";
+
+var pageLength = 50;
```

Comment:
Ah I see, I thought it was just the first number of articlse to render. Okay, something like NUM_ARTICLES_TO_SHOW or NUM_ARTICLES_IN_PAGE is fine.